webpack/webpack.dev.js

```diff
@@ -13,7 +13,7 @@ module.exports = merge(common, {
     index: "./src/js/index.js",
     vendor: Object.keys(dependencies)
   },
-  devtool: "cheap-module-eval-source-map",
+  devtool: "source-map",
```

Comment:
This seems unrelated, could you pull this into a seperate PR and open a ticket for why this is needed?

Reply:
Oops, I do this to make debug work in VSCode. Had an auto-ignore script that probably changed now that the webpack config changed and this slipped. No need for us to change this, cheap eval is faster for dev.